documentation/docs/practical-sessions/converting-synop-data-to-bufr.md

```diff
@@ -27,25 +27,30 @@ as the relationship between the information contained in the FM-12 SYNOP reports
 ## Preparation
 
 !!! warning "Prerequisites"
-    - Ensure that your wis2box has been configured and started, including the setting execution tokens 
-      for the ``processes/wis2box`` and ``collections/stations``paths. Confirm the status by visiting 
-      the wis2box API (``http://<your-host-name>/oapi``) and verifying that the API is running.
-    - The tokens can be checked by logging in to the wis2box management container and entering the 
-      command: ``wis2box auth has-access-path --path processes/wis2box <your-token>`` where 
-      ``<your-token>`` is the token you have previously entered.
-    - If the tokens are missing they can be generated with the following commands:
-    
-        ```{.copy}
-        wis2box auth add-token --path processes/wis2box <token>
-        wis2box auth add-token --path collections/stations <token>
-        ```
-      where ``<token>`` is the value of the token. This can be left blank to automatically generate
-      a random token (recommended).
-    - For practical purposes the exercises in this session use data from Romania, import the 
-      station ``0-20000-0-15015`` into your station list and associate it with the topic
-      for your "Surface weather observations collection". This will be removed at the end of the session.
+
+    - Ensure that your wis2box has been configured and started.
+    - Confirm the status by visiting the wis2box API (``http://<your-host-name>/oapi``) and verifying that the API is running.
     - Make sure that you have MQTT Explorer open and connected to your broker.
 
+In order to submit data to be processed in the wis2box-webapp you will need an auth token for "processes/wis2box".
```

Comment:
We also need the one for stations but this will have been generated previously. Do we want to add a note to this effect?